### ISSUE TITLE
Bot estimates PR size

### ISSUE THREAD

**Issue by drpaneas** [romie-gr/romie#24]
#### Description

It's good to know how big (or not) it's a PR, so we can list them and see how many of those big one (or small ones) we have in the queue.

#### Acceptance Criteria

- [ ] Install and configure https://github.com/marketplace/pull-request-size

Complexity: 1

**Comment by GrSoul** [comment]
It's a paid service.

**Comment by drpaneas** [comment]
I just got it, it's free. I've installed it; https://github.com/settings/installations/12087128

Here's what it says:

Applies size labels to each Pull Request based on the total lines of code changed.

Name | Description
-- | --
size/XS | Denotes a PR that changes 0-9 lines.
size/S | Denotes a PR that changes 10-29 lines.
size/M | Denotes a PR that changes 30-99 lines.
size/L | Denotes a PR that changes 100-499 lines.
size/XL | Denotes a PR that changes 500-999 lines.
size/XXL | Denotes a PR that changes 1000+ lines.

Permissions:
> Read access to files located at .gitattributes
> Read access to metadata
> Read and write access to pull requests

Is there anything else I need to do? I think it's ready.
